src/main/java/com/axiomq/starwars/web/controllers/FilmController.java

```diff
@@ -0,0 +1,47 @@
+package com.axiomq.starwars.web.controllers;
+
+import com.axiomq.starwars.entities.Film;
+import com.axiomq.starwars.services.FilmService;
+import com.axiomq.starwars.web.dtos.film.FilmDto;
+import com.axiomq.starwars.web.dtos.film.FilmMapper;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/film")
+public class FilmController {
+
+    private final FilmService filmService;
+
+    @PostMapping
+    public FilmDto saveFilm(@RequestBody FilmDto filmDto) {
+        return FilmMapper.INSTANCE.toDto(filmService.saveFilm(FilmMapper.INSTANCE.fromDto(filmDto)));
+    }
+
+    @GetMapping("/all")
```

Comment:
It's not necessary to add /all on the path, usually /api/film is enough for listing entities.